my_app/views.py

```diff
@@ -4,4 +4,18 @@
 
 
 def home(request):
-    return render(request, template_name='base.html')
\ No newline at end of file
+    return render(request, template_name='base.html')
+
+
+def new_check(request):
+
+    news_content = request.POST.get('news_content')
+
+    # Consultar modelo e retornar veracidade
+    veracidade = 'FALSA'
+
+    stuff_for_frontend = {
```

Comment:
little bit informal this variable name, right? 😂

Maybe `frontend_context` would be a good fit to represent what this variable does